Sources/DeepLearning/Operators.swift

```diff
@@ -250,22 +250,74 @@ public extension Tensor where Scalar: TensorFlowFloatingPoint {
     }
 
     @inlinable
-    internal func _vjpAveragePooled(
+    internal func _vjpMaxPooled3D(
+        kernelSize: (Int, Int, Int, Int, Int),
+        strides: (Int, Int, Int, Int, Int),
+        padding: Padding
+    ) -> (Tensor, (Tensor) -> Tensor) {
+        // TODO: Currently this is not higher order differentiable. Redefine in
+        // closed form.
+        let value = maxPooled3D(kernelSize: kernelSize, strides: strides,
```

Comment:
Please merge line below (total length is under 100 columns):
```suggestion
        let value = maxPooled3D(kernelSize: kernelSize, strides: strides, padding: padding)
```